src/README.md

```diff
@@ -46,15 +46,15 @@ Docker compose will still pick variables set in your shell when building and lau
 The following command will build and launch all Retail Demo Store web services in your local Docker engine.
 
 ```console
-foo@bar:~$ docker-compose up --build
+foo@bar:~$ docker compose up --build
```

Comment:
`docker-compose` and `docker compose` are different. Did you test with `docker compose` to make sure it works across all services? https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose

Reply:
yes while I was testing on a new machine/environnement, I had to install docker compose, as v1 is deprecated, installed docker compose (aka v2) and ran the whole local env with it.
There are not a lot of breaking changes see https://docs.docker.com/compose/migrate/